.github/workflows/blossom-ci.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A workflow to trigger blossom-CI on self-hosted runner
+name: blossom-ci trigger
+
+on:
+  issue_comment:
+    types: [created]
+    
+jobs:
+  build:
+    if: github.event.comment.body == 'build'
```

Comment:
Do we want this to trigger if _anyone_ says "build" or only members of the project?

On a related note, are there any plans to make the premerge CI checks a bit more automated?  It's a bit annoying to type "build" on a separate comment every time new code is pushed.  I can understand not wanting to automatically build and test arbitrary code from effectively anonymous contributors, but it seems we could do a better job automating this for project committers.

Reply:
We maintain the list of people who are authorized to trigger the build (for ngcc we submit a MR for the list, for blossom we file a ticket to blossom sre).

We could do a auto-premerge check and tell if it is triggered by internal or external contributors, I guess the idea behind it is not to burn out the internal jenkins resource until someone really need to trigger that. And yes, we can figure out a better solution later.